ge06_team04/models/stock_lot.py

```diff
@@ -0,0 +1,24 @@
+from odoo import api, fields, models
+
+
+class StockLot(models.Model):
+
+    _inherit = 'stock.lot'
+
+    name = fields.Char('Lot/Serial Number', compute='_compute_name', store=True)
+
+    @api.depends('product_id')
+    def _compute_name(self):
+        for stockLot in self:
+            product_ref = stockLot.product_id.product_tmpl_id
+            if product_ref.detailed_type == 'motorcycle' and stockLot.product_id.tracking != "none":
+                make = product_ref.make.upper() if product_ref.make else 'XX'
```

Comment:
- Use constants for `XX`
- We use % operator to format strings, e.g. "%s%s%s%s%s" % (make, model, year, battery_capacity, sequence)